pkg/apis/autoscaling/v1/conversion.go

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
Yes, please - fuzzer will not test that the metrics are correctly converted (correct struct populated).